civictechprojects/static/css/partials/_ProjectCard.scss

```diff
@@ -103,4 +103,18 @@
   float: right;
   padding-top: 4px;
   padding-right: 4px;
+}
+.ProjectCard-operation {
+  grid-row: 5;
+  grid-column: 1/5;
+  border-top: 1px solid $color-grey-medium;
+  text-align: center;
+}
+//selector specificity here is to avoid  matching .ProjectCard-root a, which is hilarious but not intended behavior
+a.ProjectCard-rsvp-btn, button.ProjectCard-rsvp-btn {
```

Comment:
It might be cleaner, longer term, to change L17 to something like  `.ProjectCard-root > a` so it only matches the anchor we want it to match, rather than having to define this level of specificity for any other links we put on a card.